test/Tests/src/Data/Text_Spec.enso

```diff
@@ -31,6 +31,11 @@ spec =
             건반(Korean).
         sentence_words = ['I', 'have', 'a', 'very', 'long', 'block', 'of', 'text', ',', 'here', '.', 'It', 'goes', 'on', 'and', 'on', ',', 'containing', 'things', 'like', 'decimal', 'points', '(', '1.0314e3', ')', 'and', 'other', 'language', 'scripts', 'as', 'well', '건반', '(', 'Korean', ')', '.']
 
+        Text.specify "should be an identity on to_text" <|
```

Comment:
```suggestion
        Text.specify "should be returned as-is by to_text" <|
```

Not sure about the original phrasing here - since the subject in the original sentence is `Text`, it reads as `Text should be an identity on to_text` - but it is the reverse - we want the `to_text function to be an identity on Text`.